src/register/register.service.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  BadRequestException,
+  HttpException,
+  HttpStatus,
+  Injectable,
+} from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import {
+  checkEmail,
+  checkGrade,
+  checkLinksToGithub,
+} from '../utils/data-validators';
+import { AddSingleRecruiterDto } from '../dtos/add-single-recruiter.dto';
+import { AddStudentsByListDto } from '../dtos/add-students-by-list.dto';
+import { hashMethod } from '../utils/hash-password';
+import { MailService } from '../mail/mail.service';
+import { UsersService } from '../users/users.service';
+import { RecruitersEntity } from '../entities/recruiters.entity';
+import { StudentsEntity } from '../entities/students-entity';
+import { UserRole, UsersEntity } from '../entities/users.entity';
+import { config } from '../config/config';
+import { randomUUID } from 'crypto';
+
+@Injectable()
+export class RegisterService {
+  constructor(
+    @InjectRepository(UsersEntity)
+    private usersRepository: Repository<UsersEntity>,
+    @InjectRepository(RecruitersEntity)
+    private recruitersRepository: Repository<RecruitersEntity>,
+    @InjectRepository(StudentsEntity)
+    private studentProfileRepository: Repository<StudentsEntity>,
+    private mailService: MailService,
+    private usersService: UsersService,
+  ) {}
+
+  async sendEmailsForUsers(data: AddStudentsByListDto[]) {
+    for (const student of data) {
+      const user = await this.usersService.findOneByEmail(student.email);
+      await this.mailService.sendMail(
+        student.email,
+        'AKTYWUJ KONTO NA PLATFORMIE HEADHUNTERS by MegaK',
+        `Aby dokończyć rejestracje na platformie HEADHUNTERS by MegaK musisz kliknąć w ten link ( localhost:3000/register/${user.id}/${user.registerToken} ) `,
```

Comment:
Kosmetyka językowa i uspójnienie.
```suggestion
        `Aby dokończyć rejestrację na platformie HEADHUNTERS by MegaK, kliknij w ten link: ${config.APP_DOMAIN}/register/${user.id}/${user.registerToken}.`,
```

Reply:
hmm poprawiałem to nie wiem czemu sie nie zapisało, ale miałem problem z utratą prawie całego dnia pracy to może coś mi umknęło podczas przywracania plików